### DISCUSSION TITLE
Scroll snap behavior is odd

### DISCUSSION THREAD

**Discussion by Coordinate-Cat** [tailwindlabs/tailwindcss#13179] [Help] [answered]
https://tailwindcss.com/docs/scroll-snap-type

Im having trouble with the scroll snap behavior.
The environment is macos and Im using the touchpad to activate inertial scrolling, but when I do so, the scrolling goes back a bit.
Does anyone know of a solution to this?

https://github.com/tailwindlabs/tailwindcss/assets/42393004/bcd0c7b5-05b7-40db-bf11-765780b2b693

**Comment by Vijaykv5** [comment, marked as answer]
Hi @Coordinate-Cat 

I think it can be fixed by adjusting the scroll-snap-align property.
Set it to "center" or "none" on the container element to ensure smoother scrolling behavior without snapping back.

**Reply by Coordinate-Cat** [reply]
Thank you @Vijaykv5 !
In my development environment, snap-y and snap-mandatory for the parent element and snap-center for the sub-element solved the problem✌️
